### PR TITLE
Styles for landscape tab

### PR DESCRIPTION
**Pull request by cazapatamar**
Styles for landscape tab

### CODE REVIEW

File: src/search/HumanFootprint.jsx [hunk 1]
```diff
@@ -9,12 +9,13 @@ import PersistenceFooprint from '../humanFootprint/PersistenceFootprint';
...
   const componentsArray = [{
     label: {
-      id: 'Huella humana actual',
+      id: 'Actual',
       name: 'Huella humana en el último año',
       disabled: false,
       expandIcon: <AddIcon />,
       detailId: 'Huella humana actual en área de consulta',
-      description: 'Huella humana identificada en el último año de medición disponible, sobre el área de consulta',
+      description: 'Huella humana identificada en el último año de medición disponible, '
```
Comment (erikasv) [inline src/search/HumanFootprint.jsx:17]: https://github.com/airbnb/javascript#strings--line-length

File: src/search/HumanFootprint.jsx [hunk 2]
```diff
@@ -41,12 +42,14 @@ const HumanFootprint = ({ generalArea, geofence }) => {
...
   {
     label: {
-      id: 'Persistencia de huella humana',
+      id: 'Persistencia',
       name: 'Persistencia',
       disabled: false,
       expandIcon: <AddIcon />,
       detailId: 'Persistencia de la huella humana en la unidad de consulta',
-      description: 'Representa la persistencia desde el origen del muestreo hasta el periodo actual, producto de análisis de huella humana en el tiempo y en esta área de consulta',
+      description: 'Representa la persistencia desde el origen del muestreo '
```
Comment (erikasv) [inline src/search/HumanFootprint.jsx:50]: https://github.com/airbnb/javascript#strings--line-length

File: src/search/HumanFootprint.jsx (hunk 1 above)
Comment (cazapatamar) [reply src/search/HumanFootprint.jsx]: Done

File: src/search/HumanFootprint.jsx (hunk 2 above)
Comment (cazapatamar) [reply src/search/HumanFootprint.jsx]: Done

**Pull request merged**